microshift_networking/microshift-firewall.adoc

```diff
@@ -22,6 +22,12 @@ include::modules/microshift-firewall-req-settings.adoc[leveloffset=+1]
 
 include::modules/microshift-firewall-opt-settings.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Currently, the customer portal is down, so I'm unable to confirm that this link works. Can you double check that it works?